src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -38,7 +38,12 @@ class GroupAutofixEndpoint(GroupEndpoint):
             RateLimitCategory.IP: RateLimit(limit=10, window=60),
             RateLimitCategory.USER: RateLimit(limit=10, window=60),
             RateLimitCategory.ORGANIZATION: RateLimit(limit=10, window=60),
-        }
+        },
+        "GET": {
+            RateLimitCategory.IP: RateLimit(limit=256, window=60),
+            RateLimitCategory.USER: RateLimit(limit=256, window=60),
+            RateLimitCategory.ORGANIZATION: RateLimit(limit=2048, window=60),
+        },
```

Comment:
tacking a rate limit on here too, verify that it should be fine with 500ms polling